### PR TITLE
[Helm] Values descriptions

### PR DESCRIPTION
**Pull request by leecalcote**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: install/kubernetes/helm/meshery/README.md [hunk 1]
```diff
@@ -16,10 +16,10 @@ Meshery chart for deploying Meshery
 |-----|------|---------|-------------|
 | affinity | object | `{}` |  |
 | annotations | object | `{}` |  |
-| env.ADAPTER_URLS | string | `"meshery-istio:10000 meshery-linkerd:10001 meshery-consul:10002 meshery-kuma:10007 meshery-osm:10009 meshery-nginx-sm:10010 meshery-nsm:10004 meshery-app-mesh:10005 meshery-traefik-mesh:10006 meshery-cilium:10012"` |  |
+| env.ADAPTER_URLS | string | `"meshery-istio:10000 meshery-linkerd:10001 meshery-consul:10002 meshery-kuma:10007 meshery-nginx-sm:10010 meshery-nsm:10004 meshery-app-mesh:10005 meshery-traefik-mesh:10006 meshery-cilium:10012"` | Optional, Meshery Server adapters with which to pre-configure Meshery Server. |
```
Comment (leecalcote) [inline install/kubernetes/helm/meshery/README.md:19]: @MUzairS15, I question whether this should even be around anymore.

File: install/kubernetes/helm/meshery/README.md [hunk 2]
```diff
@@ -16,10 +16,10 @@ Meshery chart for deploying Meshery
...
 | env.EVENT | string | `"mesheryLocal"` |  |
```
Comment (leecalcote) [inline install/kubernetes/helm/meshery/README.md:20]: @MUzairS15 do we need a GH issue to remove this antiquated env var or is it still used?

Comment (MUzairS15) [reply install/kubernetes/helm/meshery/README.md:20]: No, it is not required anymore. // @theBeginner86

File: install/kubernetes/helm/meshery/README.md (hunk 1 above)
Comment (leecalcote) [reply install/kubernetes/helm/meshery/README.md]: @theBeginner86 will you confirm as well? If not required remove in a subsequent PR, please.

**Pull request merged**